src/main/java/org/candlepin/subscriptions/tally/TallySnapshotController.java

```diff
@@ -114,9 +112,11 @@ public void produceSnapshotsForAccount(String account) {
       accountCalcs.putAll(
           retryTemplate.execute(
               context -> usageCollector.collect(this.applicableProducts, accountLookup)));
-      if (props.isCloudigradeEnabled() && null != accountCalcs.get(account)) {
-        String orgId = accountCalcs.get(account).getOwner();
+      String orgId = accountRepo.findOrgByAccountNumber(account);
+      if (props.isCloudigradeEnabled() && StringUtils.isNotEmpty(orgId)) {
         attemptCloudigradeEnrichment(account, accountCalcs, orgId);
+      } else {
+        log.info("orgId={} not found or cloudigrade is not enabled", orgId);
```

Comment:
This log message will get printed *a lot* in non-cloudigrade enabled environments.  Let's change the code to

if (props.isCloudigradeEnabled()) {
  if (StringUtils.hasLength(orgId)) {
    attemptCloudigradeEnrichment(...);
  } else {
    log.warn("Org Id {} not found for account {} during cloudigrade enrichment")
  }
}

and bump up the level to warn because if we can't find the org ID for the account, something is out of whack

Reply:
Done